source_files/edge/rad_trig.cc

```diff
@@ -339,18 +339,16 @@ rts_state_t * RAD_FindStateByLabel(rad_script_t *scr, char *label)
 // either enables them or disables them (based on `disable').
 // Actor can be NULL.
 //
-void RAD_EnableByTag(mobj_t *actor, uint32_t tag, bool disable)
+void RAD_EnableByTag(mobj_t *actor, uint32_t tag, bool disable, s_tagtype_e tagtype)

```

Comment:
Surprised to see these changes in this PR, makes me wonder about the branch

Reply:
This was a byproduct of my timing on the initial fork, but this is getting closed to favor your rebase branch now.